### PR TITLE
Adds support W3C spec exception handling.

### PR DESCRIPTION
**Pull request by staats-google**

### CODE REVIEW

File: test/sync/w3c_web_element.dart [hunk 1]
```diff
@@ -127,16 +128,22 @@ void runTests(config.createTestDriver createTestDriver) {
...
+        expect(e, new isInstanceOf<W3cWebDriverException>());
+        expect((e as W3cWebDriverException).httpStatusCode, 404);
+        expect((e as W3cWebDriverException).error, 'no such element');
+        expect((e as W3cWebDriverException).message,
+            contains('Unable to locate element'));
+      }
     });
 
-    test('findElements -- 1 found', () {
+    /*test('findElements -- 1 found', () {
```
Comment (manuelF) [inline test/sync/w3c_web_element.dart]: What's with this test?

File: test/sync/w3c_web_element.dart [hunk 2]
```diff
@@ -55,7 +56,7 @@ void runTests(config.createTestDriver createTestDriver) {
       }
       driver = null;
     });
-
+/*
```
Comment (manuelF) [inline test/sync/w3c_web_element.dart]: And this one?

File: test/sync/web_element.dart [hunk 3]
```diff
@@ -17,6 +17,7 @@ library webdriver.web_element_test;
 
 import 'package:test/test.dart';
 import 'package:webdriver/sync_core.dart';
+import 'package:webdriver/src/sync/json_wire_spec/exception.dart';
```
Comment (manuelF) [inline test/sync/web_element.dart]: Tidy

File: test/sync/w3c_web_element.dart (hunk 1 above)
Comment (staats-google) [inline test/sync/w3c_web_element.dart]: Sorry, I've not figured out to to filter tests in package test by name. There is a method using tags but I find this faster in practice.

That said, I'm seeing a benefit to using tags. :-) (Good catch.)

**Comment by staats-google** [comment]
Thanks!

**Pull request merged**